app/views/editions/secondary_nav_tabs/_admin.html.erb

```diff
@@ -1,12 +1,42 @@
 <% @edition = @resource %>
+
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds editions__admin__tab">
     <%= header_for("Admin") %>
-    <%= render "govuk_publishing_components/components/list", {
-      items: [
-        ( primary_button_for(@edition, skip_fact_check_edition_path(@edition), "Skip fact check") if @edition.fact_check? ),
-        ( link_to("Delete edition #{@edition.version_number}", confirm_destroy_edition_path(@resource), class: "govuk-link gem-link--destructive") if @edition.can_destroy? ),
-      ],
-    } %>
+
+    <% if @edition.can_create_new_edition? && @edition.published? %>
+      <p class="govuk-body">No content will be lost, but content in some fields might not make it into the new edition. If it can't be copied to a new content type it will still be available in the previous edition. Content in multiple fields might be combined into a single field.</p>
+
+      <% if @edition.respond_to?(:parts) %>
+        <p class="govuk-body">All parts of Guide Editions will be copied across. If the format you are converting to doesn't have parts, the content of all the parts will be copied into the body, with the part title displayed as a top-level sub-heading.</p>
+      <% end %>
+
+      <%= form_for @resource, url: duplicate_edition_path(@resource), method: "post" do %>
+        <%= render "govuk_publishing_components/components/radio", {
+          heading: "Update content type",
+          heading_level: 0,
+          heading_size: "s",
+          name: "to",
+          items: format_conversion_radio_options(@resource).map do |option|
+            {
+              value: option[1],
+              text: option[0],
+              hint_text: radio_options_hints[option[1]],
```

Comment:
Would it make more sense/be simpler to have `format_conversion_radio_options` return the list of items, rather than call a method that we then have to loop through and then call another method? If we do that we should probably rename the method (e.g. `conversion_items`).